content/docs/getting-started/adding-components/index.md

```diff
@@ -26,56 +38,36 @@ spec:
+<ol>
+<li> You can manually add them to Backstage using the catalog importer available at `/catalog-import`, by copying and pasting the URL of the yaml file into the importer (see video below).</li>
+<li> Backstage can automatically discover them for you. Backstages scans your Github Org for new updates periodically. Once you have added a YAML file to any repo that you want to be imported to Backstage it will be found and autopopulated as a component.</li>
+</ol>
+If you are just getting started with Backstage we recomend adding your first component by method 1, as it can be faster and will get you going quicker. 
 
-Set the owner of the component. You should be able to choose the admin group from the dropdown.
 
-![An input which allows setting the owner of the component to be imported](./preparing-pull-request-owner-component.png)
+[![Adding a Component](https://cdn.loom.com/sessions/thumbnails/faba9cbe1b154251a3c0f138e7146e41-with-play.gif)](https://www.loom.com/share/faba9cbe1b154251a3c0f138e7146e41 "Adding a Component")
 
-Click Create PR. This will open a pull request against your repository.
-
-Import the component into Backstage by clicking the Import button. You can import the component before the pull request is merged. Backstage will automatically detect when the pull request is merged.
-
-### Step 2. Merge the PR
-
-Review and merge the pull request that Backstage opens on your behalf.
-
-![An open PR on GitHub which will add a catalog-info.yaml file once merged](./opened-pr-on-github.png)
-
-### Step 3. View your component
+### Step 3. View your Component
 
 Click the Home link in the Backstage sidebar to go back to the catalog where you should see your component. Depending on the type of component you imported, you might have to cycle through the tabs until you see your component.
 
-![our component visible on the other tab of Backstage](./component-on-other-tab.png)
+![See your new component](./ViewComponents.png)
 
-## Configuring Backstage to auto discover github repositories using the github discovery processor
-
-The Roadie team can configure your Backstage instance to auto discover `catalog-info.yaml` files created in your GitHub repositories. To do this, please contact us via Slack and we will be happy to configure it for you.
-
-We will need a URL pattern to discover the catalog files.
-
-```text
-https://github.com/AcmeInc/*/blob/main/catalog-info.yaml
-```
+### TroubleShooting FAQs
+**Component Not Appearing?** </br>
+If your component is not appearing make sure Backstage has permissions to the repo that you added the yaml file too. 
+You can check this by going to the Github settings of a repo that Backstage already has access to, then follow `Settings>Integrations>Configure`, and making sure your repo is listed in the allowed repos:
 
-You can use regular regex syntax in the repository name part of the GitHub URL as shown above.
+![repo permissions](./repopermissions.png)
```

Comment:
Image not working.

Reply:
I added the image file and updated the typo in the file name, but it still isn't working. I can't figure it out ☹️

Reply:
tried it in lowercase too, didn't work

Reply:
I think its the capitals in the file names themselves. I added files without caps here: https://github.com/RoadieHQ/marketing-site/pull/374 and https://github.com/RoadieHQ/marketing-site/pull/373

Reply:
Once they are reviewed we'll see if it works.